models/docs/cli/migration.rst

```diff
@@ -0,0 +1,66 @@
+.. _other-migration:
+
+##########################
+ Migrating your checkpoint
+##########################
+
+anemoi-models provides a way for users to migrate old checkpoints so that they can be
+used with recent versions of anemoi-models.
+
+********
+ Inspect
+********
+
+To inspect the registered versions in the checkpoint, run:
+
+.. code:: bash
+
+   anemoi-models migration inspect PATH_TO_CKPT
+
+You can remove colors with the ``--no-color`` argument.
+
+
+********
+ Migrate
+********
+
+If you want to use your checkpoint with the currently installed version of anemoi-models,
+you can use:
+
+.. code:: bash
+
+   anemoi-models migration sync PATH_TO_CKPT
```

Comment:
Not related strictly to the PR but the CLI in training also have some utilities related to checkpoints - https://github.com/ecmwf/anemoi-core/blob/main/training/src/anemoi/training/commands/checkpoint.py to convert a training checkpoint to an inference one. It could be good in the future to think about unifying these two CLIs (note the training conversion requires pytorch lightning so that's why it lives in anemoi-training and not in models)